docs/middlewares.md

```diff
@@ -140,6 +141,104 @@ handler(event, context, (_, response) => {
 })
 ```
 
+## [functionShield](/src/middlewares/functionShield.js)
+
+Hardens AWS Lambda execution environment:
+* By monitoring (or blocking) outbound network traffic from your function, you can be certain that your data is never leaked
+* By disabling read/write operations on the /tmp/ directory, you can make your function truly ephemeral
```

Comment:
`you can make your function truly ephemeral` is a bold claim, and untrue (there's all kinds of other things I can do in my code to make my function stateful), and more importantly it's not all that important to me as a developer.

Being ephemeral is means to an end, and in this particular case I'd be more incentivized to use this feature if you explain why using /tmp folder is a bad idea, and why it should be disabled, maybe pointing to an article?